SpatialGDK/Source/SpatialGDK/Private/Utils/EntityPool.cpp

```diff
@@ -78,6 +79,7 @@ void UEntityPool::ReserveEntityIDs(int32 EntitiesToReserve)
 		if (!bIsReady)
 		{
 			bIsReady = true;
+			NetDriver->Sender->CreateServerWorkerEntity(GetNextEntityId());
```

Comment:
The coupling here seems a little much - could we make this a delegate instead?

Reply:
yep sgtm will do